nodule.js

```diff
@@ -82,13 +84,21 @@ module.exports = function(app, config) {
     // nodules can have multiple routes
     var routeArray = (typeof seedNodule.route === 'string') || (seedNodule.route instanceof RegExp) ? [seedNodule.route] : seedNodule.route;
     _.each(routeArray, function(routePath) {
-      seedNodules[routePath] = seedNodule; // routes must me unique
-        
+
+      // throw error if duplicate route found in index
+      var routeLookup = seedNodule.routeVerb.toLowerCase() + '_' + routePath.toLowerCase();
+      if (seedNodules[routeLookup]) {
+        throw new Error('Duplicate Route Matching: ' + seedNodules[routeLookup].path + ' // ' + seedNodules[routeLookup].name);
```

Comment:
In express 4 - you call *next('route')* and it throws the request back into the route chain (starting with where it left off, not at the top).